.github/workflows/ci.yml

```diff
@@ -164,7 +164,9 @@ jobs:
         brew install ace boost cmake eigen gsl ipopt jpeg libedit opencv pkg-config qt5 sqlite swig tinyxml
         # ROBOTOLOGY_ENABLE_DYNAMICS dependencies 
         brew install libmatio
-        # ROBOTOLOGY_USES_GAZEBO dependencies 
+        # ROBOTOLOGY_USES_GAZEBO dependencies
+        # Workaround for https://github.com/robotology/robotology-superbuild/issues/479
+        brew tap https://github.com/robotology/robotology-superbuild/issues/479
```

Comment:
Why there is the url of the issue?

Reply:
Because it is a typo, thanks for noticing.